app/styles/_library/_base.forms.scss

```diff
@@ -54,10 +54,12 @@ input[type=checkbox] {
   cursor: pointer;
   display: block;
   float: left;
-  border: 2px solid rgb(var(--color-border-bold));
+  border: 2px solid rgb(var(--color-gray), 0.3);
   padding: 0;
   user-select: none;
   appearance: none;
+  -webkit-appearance: none;
```

Comment:
seems like there should be a mixin to do this

Reply:
We should probably do a styles cleanup at one point, go through an optimize things. I'll make a note to do a ticket for this so we don't forget!